core-standards.php

```diff
@@ -2,7 +2,7 @@
 
 /*
   Plugin Name: Core Standards
-  Version: 1.24.0
+  Version: 2.0.0
```

Comment:
Which incompatibility is introduced by this code?

Reply:
@fabianmarz may need to answer that

Reply:
The cookie consent changes from a simple banner to a modal window and thus can change the expected user behaviour.